web-client/src/presenter/computeds/trialSessionDetailsHelper.js

```diff
@@ -1,19 +1,46 @@
+import {
```

Comment:
🔧 I think entity constants in the web-client should be pulled off of `applicationContext.getConstants()` instead of directly imported

Reply:
Done